schema/echo.proto

```diff
@@ -36,7 +36,7 @@ option (google.api.metadata) = {
 service Echo {
   // This service is meant to only run locally on the port 7469 (keypad digits
   // for "show").
-  option (google.api.default_host) = "localhost:7469";
+  option (google.api.default_host) = "localhost";
```

Comment:
Why this change? We do not have a port in the annotations, so this is needed, yes?